apps/indexer/src/api/mappers/proposals/proposals.ts

```diff
@@ -29,6 +29,14 @@ export const ProposalsRequestSchema = z.object({
     }),
   fromDate: z.coerce.number().optional(),
   fromEndDate: z.coerce.number().optional(),
+  proposalType: z
+    .union([z.coerce.number(), z.array(z.coerce.number())])
+    .optional()
+    .transform((val) => {
+      if (!val) return undefined;
+      // Always normalize to array
+      return typeof val === "number" ? [val] : val;
+    }),
```

Comment:
could this be an ENUM type instead of a ✨ magic number ✨?

Reply:
now it is coming from the dao_config. Each DAO can have its own number that represents the optimisticProposal